exercises/concept/welcome-to-tech-palace/welcome_to_tech_palace_test.go

```diff
@@ -0,0 +1,102 @@
+package techpalace
```

Comment:
I am not certain if this conforms to our current test guidance for the go track. The top-level README.md file suggests these should use t.Fatalf rather than t.Errorf. Perhaps that is out of date?

There also seems to be a mild preference for using `got` and `want` rather than `expected`.

@ekingery?

Reply:
I'm pretty sure we do want `t.Fatalf`, so that the test runner stops after the first failure. Although I'm going from memory and I think @ErikSchierboom wrote the test runner spec / docs so I'll leave that up to him.

Yes, swapping out `expected` with `want` would make it more consistent with the docs and other tests across the track. Thanks!

Reply:
> I'm pretty sure we do want t.Fatalf, so that the test runner stops after the first failure. Although I'm going from memory and I think @ErikSchierboom wrote the test runner spec / docs so I'll leave that up to him.

I think all tests should run so we can list the passed/failed numbers to a student.

Reply:
> I'm pretty sure we do want t.Fatalf, so that the test runner stops after the first failure. Although I'm going from memory and I think @ErikSchierboom wrote the test runner spec / docs so I'll leave that up to him.

We used to only display the first error, but have since reverted that decision:

![image](https://user-images.githubusercontent.com/135246/134660267-901b6cf7-7a2b-4bda-8d7a-fd0f7e9e241d.png)

> Yes, swapping out expected with want would make it more consistent with the docs and other tests across the track. Thanks!

Done. It might be worth opening an issue to convert other exercises too.